tavern_pathfinder/Cargo.toml

```diff
@@ -22,3 +22,5 @@ maintenance = { status = "actively-developed" }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+serde = {Version = "1.0.106", features = ["derive"]}
+uuid = {Version = "0.2", features = ["serde"]}
```

Comment:
The version tag needs to be all lowercase. Compiling gives warnings and our grcov test coverage action fails, I believe because of this.